diesel_cli/src/infer_schema_internals/data_structures.rs

```diff
@@ -79,7 +79,10 @@ impl ColumnType {
     }
 }
 
-use std::fmt;
+use std::{
```

Comment:
Minor stylistic nit: Diesel we use unnested imports in all other parts of the code base, so I would like to keep this consistent.